src/lib/constants.js

```diff
@@ -14,12 +14,16 @@ const constants = {
   "REQUEST_SEARCH_SUGGESTIONS",
   "RECEIVE_SEARCH_SUGGESTIONS",
   "UPDATE_SEARCH_STRING",
+  "RECEIVE_UPDATE_SITES",
   "REQUEST_CURRENT_SEARCH_ENGINE",
   "RECEIVE_CURRENT_SEARCH_ENGINE",
   "REQUEST_VISIBLE_SEARCH_ENGINES",
   "RECEIVE_VISIBLE_SEARCH_ENGINES",
   "REQUEST_FRECENT",
-  "RECEIVE_FRECENT"
+  "RECEIVE_FRECENT",
+  "REQUEST_INIT_USER_DATABASE",
+  "REQUEST_INIT_USER_DATABASE_COMPLETE",
+  "REQUEST_UNDO_DIALOG_VISIBILITY"
 ].forEach(action => constants[action] = action);
```

Comment:
BTW, is this a "React-ism"? Seems kinda strange to declare an object whose name/values are the same 0_o. That is, if `c.REQUEST_UNDO_DIALOG_VISIBILITY === "REQUEST_UNDO_DIALOG_VISIBILITY"` then just inline `"REQUEST_UNDO_DIALOG_VISIBILITY"`, and friends and we save a redundant import. 

Or am I missing something?